Bpor.Rms/Bpor.Rms.Infrastructure/Bpor.Rms.Infrastructure.csproj

```diff
@@ -8,9 +8,11 @@
 
     <ItemGroup>
       <PackageReference Include="AWSSDK.SecretsManager" Version="3.7.302.30" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.3" />
       <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="8.0.0" />
       <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
       <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
+      <PackageReference Include="MySqlConnector" Version="2.3.5" />
```

Comment:
I'm not familiar with this package. Is this going to bind our database schema to objects in C# in conjunction with Entity Framework?